src/components/NewTask/NewTask.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './NewTask.css';
+
+class NewTask extends React.Component {
+  state = {
+    newTaskText: '',
+    userOption: '',
+    errors: {
+      title: false,
+      username: false,
+    },
+  };
+
+  handleChange = (event) => {
```

Comment:
мы не используем сам event, упростим?
```suggestion
  handleChange = ({ target }) => {
```